src/sdk/pynni/nni/nas/tensorflow/mutables.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+
+from tensorflow.keras import Model
+
+
+_logger = logging.getLogger(__name__)
+
+_counter = 0
+
+def _global_mutable_counting():
```

Comment:
this counting makes it hard to do key matching of two independent running. For example, one is training and the other is applying.